docs/nuget-sdk.md

```diff
@@ -0,0 +1,85 @@
+# NuGet SDK
+
+NuGet is primarily a tool, but we also publish packages that others can reference in their own projects. All of these packages are published to NuGet.org with the [`NuGet.` prefix](https://www.nuget.org/packages?q=nuget.*). Documentation on the NuGet SDK is on our docs site [https://docs.microsoft.com/nuget/reference/nuget-client-sdk](https://docs.microsoft.com/nuget/reference/nuget-client-sdk)
+
+The projects that are part of the SDK are largely under `src\NuGet.Core`, but also include `NuGet.VisualStudio` and `NuGet.VisualStudio.Contracts` under `src\NuGet.Clients`, which define the interfaces for our Visual Studio extensibility APIs.
```

Comment:
Do all projects under `src\NuGet.Core` are part of the SDK?

Reply:
Depends on how strictly you define the SDK. They all get shipped as packages. However, one of them is a command line app, not a library, it just happens to be packed as a library because of how we insert it into the .NET SDK. At least two projects ship MSBuild props/targets/tasks. Another project ships a command line program used by one of the msbuild targets project.

So, they're not all packages that are useful for other people to reference in their own projects, but they are all packed into nupkgs.